bin/mops-groups

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var fs = require('fs');
+var program = require('commander');
+var MMS = require('node-mms-client');
+var MopsConfig = require('../lib/util/MopsConfig');
+
+var config = new MopsConfig();
+var mms = new MMS({
+	username: config.get('user'),
+	apiKey: config.get('apiKey'),
+	host: config.get('host')
+});
+
+program
+  .option('-l, --list', 'show all groups visible to this user')
+  .option('-s, --setDefault <groupName>', 'set default group for API requests')
+  .option('-d, --delete <groupName>', 'delete group')
+  .parse(process.argv);
+
+var outputList = function() {
+    mms.groups().list().then(
+        function(resp) {
+            console.log(resp.results);
```

Comment:
[np] Pete was using JSON.stringify to pretty print. 
`console.log(JSON.stringify(alertsConfig, null, 4));`